README.md

```diff
@@ -160,6 +160,29 @@ addServiceWorker({
 ```
 
 
+### Generating an HTTP/2 Push Manifest
+
+`polymer-build` can automatically generate a [push manifest](https://github.com/GoogleChrome/http2-push-manifest) for your application. This JSON file can be read by your HTTP/2 push-enabled web server to more easily construct the appropriate `Link: <URL>; rel=preload; as=<TYPE>` headers(s) for HTTP/2 push/preload.
```

Comment:
Might be good to drop in a link to one of the sample servers that support this. It'd be my first question.